tools/tools.go

```diff
@@ -0,0 +1,8 @@
+//+build tools
+
+package tools
+
+import (
+	// pins update-docker-tags CLI
+	_ "github.com/slimsag/update-docker-tags"
+)
```

Comment:
Do we actually need to pin this CLI in this repo? I'm not going to change it without accounting for Sourcegraph, so it seems fine to not pin it. Given this repo is meant to be documentation/reference (see README.md) it would be nice to keep these files (tools/tools.go, go.sum, go.mod) out of this repo all together

Reply:
@davejrt was unable to update the tags through the release tool during the last release because his version of this tool did not have my patch adding `-enforce` - I think it is generally good practice to have managing tool versions be a bit seamless where possible, since even if there is documentation, being faced with an error that's related to an out-of-date tool is usually the most frustrating experience ever 😓